package.json

```diff
@@ -24,7 +24,7 @@
     "start": "pm2-runtime start pm2.config.js",
     "heroku-postbuild": "echo 'Skipping build'",
     "build": "node scripts/prod.js",
-    "clean": "rm -rf ./packages/client/__generated__ ./queryMap.json ./packages/server/graphql/public/schema.graphql ./packages/server/graphql/private/schema.graphql ./dev ./dist ./build",
+    "clean": "git clean -fdx -e .env",
```

Comment:
+1 I might be an exception, but I often have untracked files from stuff I tried or when I've run `createJiraServerKeys` from inside the project folder. Maybe consider renaming it to `yarn deepclean`?
Apart from that, it does not clean the databases, should we do this separately? Something along the lines of
```
yarn db:stop && docker volume rm docker_postgres-data docker_rethink-data && yarn db:start
```

Reply:
I want to keep db stuff separately, as it's technically a different thing. I have a draft PR here: https://github.com/ParabolInc/parabol/pull/8010. what I'm working on, is being able to group all parabol related images so it's easier to see that in docker-mac app/cli and perform all the cleanups.

Reply:
I personally would expect only generated files to be deleted (otherwise I would run `git clean -xfd` manually). I'm not the target audience though, so I'm totally fine with it as is.

Reply:
I share the concern and appreciate the feedback! what about this
```
    "clean": "git clean -fdX -e .env", // removes everything that's ignored except for the .env file
    "fresh": "git clean -fdx -e .env", // removes everything that's ignored and untracked, except for the .env file
```

Reply:
There's a `-e`? 🤦 . That makes it easy!
Yeah, LGTM

Reply:
@BartoszJarocki I like your proposal of having `yarn clean` only cleaning ignored files